documentation/guides/migrating-from-v3-to-v4.md

```diff
@@ -0,0 +1,390 @@
+# Migrating from v3 to v4
+
+Polaris v4 contains mostly internal changes and has simple migration path. This guide goes through the necessary steps to deal with breaking changes in your application.
```

Comment:
```suggestion
Polaris v4 contains mostly internal changes and has one migration path. This guide goes through the necessary steps to deal with breaking changes in your application.
```

Reply:
We avoid using the term "simple" because if a newer developer finds the task challenging they may feel frustrated that they didn't find it "simple". We're making an assumption when we use the term. Does "one" migration path work to communicate the message?

Reply:
```suggestion
Polaris v4 contains mostly internal changes and has many migration paths. This guide goes through the necessary steps to deal with breaking changes in your application.
```